packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -266,15 +264,14 @@ export const removeItemFromCart = payload => {
 
             // check if the guest cart has expired
             if (noGuestCartId || (response && response.status === 404)) {
-                // if so, then delete the cached ID...
-                // in contrast to the save, make sure storage deletion is
+                // Delete the cached ID from local storage.
+                // The reducer handles clearing out the bad ID from Redux.
```

Comment:
`removeItemFromCart` is a little different in that the reducer handles clearing out the bad ID from redux, so we don't have to do it here.